.github/workflows/build.yaml

```diff
@@ -8,3 +8,17 @@ jobs:
       - uses: actions/checkout@v1
       - name: Build the Docker image
         run: docker build .
+      - name: Set up Go
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14.x
+      - name: Build release snapshot
+        run: make snapshot
+      - name: Archive release snapshot
+        uses: actions/upload-artifact@v2
```

Comment:
I'm not familiar with `actions/upload-artifact` - what will this do?

Reply:
It uploads files as artifacts to the workflow run page. From there you can download them. It's very useful to see the results without making a full release. 

For more information see https://github.com/actions/upload-artifact

Reply:
You can see it in action at https://github.com/ribbybibby/ssl_exporter/actions/runs/205589666, it appears in the Artifacts section:

![image](https://user-images.githubusercontent.com/43356/90156517-75343a00-dd84-11ea-9f9f-dff7b6f2c02e.png)